packages/playground/src/components/select_vm_image.vue

```diff
@@ -29,13 +29,11 @@
       </input-tooltip>
     </input-validator>
 
-    <input-validator :rules="[validators.required('Entry point is required.')]" :value="entryPoint" #="{ props }">
-      <input-tooltip
-        tooltip="The entry point of the selected flist. It's the first process that runs on the machine once it's deployed."
-      >
-        <v-text-field label="Entry Point" v-model="entryPoint" v-bind="props" />
-      </input-tooltip>
-    </input-validator>
+    <input-tooltip
+      tooltip="The entry point of the selected flist. It's the first process that runs on the machine once it's deployed."
+    >
+      <v-text-field label="Entry Point" v-model="entryPoint" v-bind="props" />
+    </input-tooltip>
```

Comment:
@maayarosama here why did we remove validation ? also if removing validation is needed why we still have `v-bind="props"` on line #35 ?

Reply:
Entrypoint isn't required anymore, will remove ```v-bind="props"```